index.yaml

```diff
@@ -1,13 +1,40 @@
 apiVersion: v1
 entries:
   service-binding-operator:
+  - annotations:
+      charts.openshift.io/name: service-binding-operator
+      charts.openshift.io/provider: RedHat
+      charts.openshift.io/supportURL: https://github.com/redhat-developer/service-binding-operator.git
+    apiVersion: v2
+    appVersion: 1.2.0
+    created: "2022-09-13T20:23:46.992831+05:30"
+    description: A Helm chart to deploy service binding operator
+    digest: a4f9cd2121972985f89ecf1d06b7a75355b5aa7d26d0f7ad1ce72bf2691810c1
+    icon: https://raw.githubusercontent.com/redhat-developer/service-binding-operator/master/assets/icon/sbo-logo.png
+    keywords:
+    - ""
+    kubeVersion: '>= 1.19.2-0'
+    maintainers:
+    - email: kmamgain@redhat.com
+      name: Kartikey-star
+    - email: pmacik@redhat.com
+      name: pmacik
+    - email: jchui@redhat.com
+      name: jasperchui
+    - email: dperaza@redhat.com
+      name: dperaza4dustbit
+    name: service-binding-operator
+    type: application
+    urls:
+    - service-binding-operator-1.1.0.tgz
```

Comment:
I think it is good, but just for me to understand, what is the criteria to move this version to patch , minor or mayor

Reply:
I think since we have changed the template/test and added a param to values.yaml this should be a minor changes , so 1.1.0. Previously there was no major change on the chart, only the image hash in the deployment image changed.
When sbo moves to 2.x.x. we should move as well considering that will be a major change.
Going forward for the sake of consistency i would say we should  keep the app version and chart version same.